src/tracing/errors.ts

```diff
@@ -4,19 +4,22 @@ import { ClientOptions, ErrorEvent, EventHint } from '@sentry/types'
 function isEthersRequestError(error: Error): error is Error & { requestBody: string } {
   return 'requestBody' in error && typeof (error as unknown as Record<'requestBody', unknown>).requestBody === 'string'
 }
-
 /**
  * Filters known (ignorable) errors out before sending them to Sentry.
  * Intended as a {@link ClientOptions.beforeSend} callback. Returning null filters the error from Sentry.
  */
 export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event: ErrorEvent, hint: EventHint) => {
   const error = hint.originalException
   if (error instanceof Error) {
-    // ethers aggressively polls for block number, and it sometimes fails (whether spuriously or through rate-limiting).
-    // If block number polling, it should not be considered an exception.
     if (isEthersRequestError(error)) {
       const method = JSON.parse(error.requestBody).method
+      // ethers aggressively polls for block number, and it sometimes fails (whether spuriously or through rate-limiting).
+      // If block number polling fails, it should not be considered an exception.
       if (method === 'eth_blockNumber') return null
+
+      // For now, these errors are not actionable so we should not report them.
+      // ethers exceptions are currently not caught in the codebase because we don't try/catch all provider calls.
+      if (method.startsWith('collect(')) return null
```

Comment:
`collect` here seems to be referring to a specific onchain interaction for LPs ([link](https://gist.github.com/Aboudjem/886f9fae8649d3801b4ffe6e9bac1564#file-uniswapinterfaces-sol-L1211) ) so maybe update the comment to reflect that, or use a more generic string match .

Reply:
updated to use CALL_EXCEPTION code, but we may be better off fixing the specific error here instead?